package arrayafter;

public class OwnException extends Throwable {

	OwnException(String s)
	{
System.out.println(s);	}
}
